libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html

```diff
@@ -3,4 +3,6 @@
 
   <elewa-group-home-hero-section></elewa-group-home-hero-section>
   
-</div>
\ No newline at end of file
+</div>
+
+<!-- <app-elewa-group-card></app-elewa-group-card> -->
```

Comment:
This change was not neccesary

Reply:
This happened after pulling changes from main branch